pkg/controller/useraccount/useraccount_controller.go

```diff
@@ -608,3 +644,59 @@ func newNSTemplateSet(userAcc *toolchainv1alpha1.UserAccount) *toolchainv1alpha1
 func ToIdentityName(userID string, identityProvider IdentityProvider) string {
 	return fmt.Sprintf("%s:%s", identityProvider.GetIdP(), userID)
 }
+
+func (r *ReconcileUserAccount) lookupAndDeleteCheUser(userAcc *toolchainv1alpha1.UserAccount) error {
+
+	// If the admin username is not set then Che user deletion is not configured, just return
+	if r.config.GetCheAdminUsername() == "" {
+		log.Info("Che user deletion is not configured, configure the Che admin credentials to enable it")
+		return nil
+	}
+
+	var userExists bool
```

Comment:
this line is not needed, AFAICT

Reply:
You're right, good catch! I did a bit of refactoring so must have left it out. Done